File: LeetCode/LeetCode 75/1431. Kids With the Greatest Number of Candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> ans = new ArrayList<Boolean>();
        int maxCandy = -1;
        for(int i = 0; i<candies.length; i++) {
            if(candies[i] >= maxCandy) {
                maxCandy = candies[i];
            }
        }

        for(int i = 0; i<candies.length; i++) {
            if(candies[i]+extraCandies >= maxCandy) {
                ans.add(true);
            }
            else{
                ans.add(false);
            }
        }
        return ans;
    }
}

File: Leetcode/Leetcode 75/1431. Kids With the Greatest Number of Candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    let maxVal = Math.max(...candies);
    let ans = [];
    for (num of candies) {
        ans.push(num + extraCandies >= maxVal);
    }
    return ans;
};
